python/py-scikit-learn/Portfile

```diff
@@ -46,6 +46,7 @@ if {${name} ne ${subport}} {
                             sha256  c503802a81de18b8b4d40d069f5e363795ee44b1605f38bc104160ca3bfe2c41 \
                             size    11818490
     } else {
+        compiler.openmp_version 2.5
```

Comment:
A few months ago, I had asked on Gitter whether OpenMP 2.5 is sufficient, and was told that it probably is: https://gitter.im/scikit-learn/scikit-learn?at=5d0e97a21e35ef14b69ec123

Does this appear to be correct?